### PR TITLE
Add NCC security assessment announcement blog

### PR DESCRIPTION
**Pull request by nrjpoddar**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by nrjpoddar** [comment]
cc @craigbox

**Comment by nrjpoddar** [comment]
The blog has been extensively reviewed in private Google doc and approved by all stake holders including PS WG and Google comms team.

File: content/en/blog/2021/ncc-security-assessment/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+title: "Announcing the results of Istio’s first security assessment"
+description: Results of NCC security assessment.
+publishdate: 2021-07-13
+attribution: "Neeraj Poddar (Aspen Mesh), on behalf of Istio Product Security Working Group"
+keywords: [istio,security,audit,ncc,assessment]
+---
+
+Istio service mesh has gained wide production adoption across a wide variety of
```
Comment (craigbox) [inline content/en/blog/2021/ncc-security-assessment/index.md:9]:
```suggestion
The Istio service mesh has gained wide production adoption across a wide variety of
```

**Review by craigbox — CHANGES_REQUESTED** [review]

**Review by craigbox — APPROVED** [review]

**Pull request merged**